plugins/inputs/gnmi/README.md

```diff
@@ -64,10 +64,22 @@ It has been optimized to support gNMI telemetry as produced by Cisco IOS XR (64-
 
     ## If suppression is enabled, send updates at least every X seconds anyway
     # heartbeat_interval = "60s"
+
+  [[inputs.gnmi.subscription]]
+   name = "descr"
+   origin = "openconfig-interfaces"
+   path = "/interfaces/interface/state/description"
+   subscription_mode = "on_change"
+
```

Comment:
Usually we leave optional configuration commented out. Make sure the indentation matches what's already here